packages/core/src/card/card.element.scss

```diff
@@ -0,0 +1,18 @@
+@import './../styles/tokens/generated/index';
+
+:host {
+  --width: auto;
+  --height: auto;
+
+  --background: #{$cds-alias-object-container-background};
+  --border: #{$cds-alias-object-border-width-100} solid #{$cds-alias-object-container-border-color};
+  --border-radius: #{$cds-alias-object-border-radius-100};
+  --box-shadow: #{$cds-alias-object-shadow-100};
+}
+
+/**
+ * TODO: Have to figure out why is not working the `:not` selector to `cds-divider` component
```

Comment:
You may have an issue here due to those elements (the `cds-divider` and its containing `div`) being slotted...